libs/auth-api-lib/src/lib/resources/delegation-resources.service.ts

```diff
@@ -388,4 +361,28 @@ export class DelegationResourcesService {
     }
     return false
   }
+
+  private async findCustomDelegationScopes(user: User): Promise<string[]> {
```

Comment:
~ Rename to indicate that this only looks up for actor, otherwise returns empty `[]`
```suggestion
  private async findActorCustomDelegationScopes(user: User): Promise<string[]> {
```